components/movementsensor/gpsnmea/pmtkI2C.go

```diff
@@ -243,10 +243,17 @@ func (g *PmtkI2CNMEAMovementSensor) Position(ctx context.Context, extra map[stri
 }
 
 // Accuracy returns the accuracy, hDOP and vDOP.
-func (g *PmtkI2CNMEAMovementSensor) Accuracy(ctx context.Context, extra map[string]interface{}) (map[string]float32, error) {
+func (g *PmtkI2CNMEAMovementSensor) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
 	g.mu.RLock()
 	defer g.mu.RUnlock()
-	return map[string]float32{"hDOP": float32(g.data.HDOP), "vDOP": float32(g.data.HDOP)}, g.err.Get()
+	acc := movementsensor.Accuracy{
+		AccuracyMap:        map[string]float32{"hDOP": float32(g.data.HDOP), "vDOP": float32(g.data.VDOP)},
+		Hdop:               float32(g.data.HDOP),
+		Vdop:               float32(g.data.VDOP),
+		NmeaFix:            int32(g.data.FixQuality),
```

Comment:
note to Susmita - this data struct is your way of getting rid of `ReadFix` in the gpsnmea driver. Everything in Gps-rtk should be gotten off the GPS parser's data struct.